spec/features/user_cant_sign_in_spec.rb

```diff
@@ -83,6 +83,45 @@
 
       screenshot!
     end
+
+    scenario "user tries to login with unconfirmed email" do
```

Comment:
This test is a dupe of https://github.com/openstax/accounts/blob/ff73ded343e11f578a3bfbf22436b9c6dbbe104a/spec/features/user_cant_sign_in_spec.rb#L10